src/fontra/client/web-components/range-slider.js

```diff
@@ -162,169 +160,172 @@ export class RangeSlider extends LitElement {
     this.maxValue = 100;
     this.defaultValue = this.minValue;
     this.value = this.defaultValue;
-    this.tickmarksPositions = [];
-    this.step = "any";
+    this.step = 1;
     this.sawMouseDown = false;
     this.sawMouseUp = false;
     this.onChangeCallback = () => {};
   }
 
-  render() {
-    delete this._rangeInputElement;
+  get valueFormatted() {
     const minMaxRange = this.maxValue - this.minValue;
     const decimalPlaces = minMaxRange < 100 ? 3 : 2;
-    const value = round(this.value, decimalPlaces);
-    const minValue = round(this.minValue, decimalPlaces);
-    const defaultValue = round(this.defaultValue, decimalPlaces);
-    const maxValue = round(this.maxValue, decimalPlaces);
-    const isAtDefault = this.value == this.defaultValue;
-    this.updateIsAtDefault();
-    return html`
-      <section class="wrapper">
-        <div class="numeric-input">
-          <section class="slider-input">
-            <input
-              type="number"
-              @change=${this.changeValue}
-              @keydown=${this.handleKeyDown}
-              class="slider-numeric-input"
-              min=${this.minValue}
-              max=${this.maxValue}
-              step=${this.step}
-              pattern="[0-9]+"
-              .value=${value}
-            />
-          </section>
-        </div>
-        <div class="range-container">
-          <input
-            type="range"
-            @input=${this.changeValue}
-            @change=${this.handleChange}
-            @mousedown=${this.handleMouseDown}
-            @keydown=${this.handleKeyDown}
-            @mouseup=${this.handleMouseUp}
-            class="slider ${isAtDefault ? "is-at-default" : ""}"
-            min=${this.minValue}
-            max=${this.maxValue}
-            step=${this.step}
-            .value=${this.value}
-            list="markers"
-            tabindex="-1"
-          />
-        </div>
-      </section>
-    `;
+    return round(this.value, decimalPlaces);
+  }
+
+  set value(value) {
+    this._value = value;
+    if (this.rangeInput) this.rangeInput.value = value;
```

Comment:
Please no one-line if statements: use a proper { } block. Same for the line below.

(We need to check if there's a prettier option for that)